vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -1983,34 +1983,67 @@ private boolean sendMessageByInvitingLeftMemberInDirectChat(final @Nullable Inte
         // In the case of a direct chat, we check if the other member has left the room.
         String leftMemberId = leftMemberInDirectChat();
         if (null != leftMemberId) {
-            Log.d(LOG_TAG, "sendMessageByInvitingLeftMemberInDirectChat: invite again " + leftMemberId);
-            mRoom.invite(leftMemberId, new ApiCallback<Void>() {
+            // Check whether the left member did not deactivate his account
+            mSession.getProfileApiClient().displayname(leftMemberId, new ApiCallback<String>() {
```

Comment:
Do you display a loading wheel here?

Reply:
No, we don't
I noticed that we didn't display loading wheel for the invite too, so I decided to do the same for the moment